static/elements/chromedash-process-overview.js

```diff
@@ -198,6 +235,20 @@ class ChromedashProcessOverview extends LitElement {
       </div>`;
   }
 
+  willUpdate() {
```

Comment:
This code looks ok, but the name `willUpdate` looks like you are saying that this is a `TODO`.  I don't see what calls this.

Reply:
willUpdate is part of the lit element lifecycle, called before update(), which calls render().  So this functions the same as putting the code at the top of the render function, but it seems better in willUpdate, since it is not rendering anything.  See https://lit.dev/docs/components/lifecycle/#reactive-update-cycle-performing

Reply:
OK, I had not noticed that before.